types/twitter_cldr/index.d.ts

```diff
@@ -0,0 +1,173 @@
+// Type definitions for twitter_cldr 2.2
+// Project: https://github.com/twitter/twitter-cldr-npm/
+// Definitions by: lionel-rowe <https://github.com/DefinitelyTyped>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.9
```

Comment:
```suggestion
```
no longer enforced here (see timeline support window in the readme doc)

Reply:
Removing it causes errors when I run `npm test twitter_cldr` locally:

```log
ERROR: 6:18   expect  Compile error in typescript@3.8 but not in typescript@3.9.
Fix with a comment '// Minimum TypeScript Version: 3.9' just under the header.
Argument of type '"???"' is not assignable to parameter of type 'Locale'.
ERROR: 32:34  expect  Compile error in typescript@3.8 but not in typescript@3.9.
Fix with a comment '// Minimum TypeScript Version: 3.9' just under the header.
Argument of type '"???"' is not assignable to parameter of type 'RbnfGroupName'.
ERROR: 47:39  expect  Compile error in typescript@3.8 but not in typescript@3.9.
Fix with a comment '// Minimum TypeScript Version: 3.9' just under the header.
Type '"???"' is not assignable to type '"LTR" | "RTL"'.
```

This is presumably because `@ts-expect-error` isn't recognized in older versions. Another fix would be replacing those with `$ExpectError`, but the docs recommend `@ts-expect-error` instead:

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/04d5ce209125122ab55ad7502c88a5ef05c1b9a8/README.md?plain=1#L272

Reply:
can you verify you've branched from latest version of DT repo? I cannot see this error locally

Reply:
As with https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63687, `rm -r node_modules ; npm i` fixed it.